streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -162,10 +164,12 @@ public int hashCode() {
 
     public StreamsMetricsImpl(final Metrics metrics,
                               final String clientId,
+                              final String processId,
```

Comment:
I know this might mean be a big splash radius for this PR, but should we pass in an UUID instead of a String, and just call `processId.toString()` when putting it into the `tagMap` ?

Reply:
Yeah, this would have a large splash radius. I'd rather leave it as is, not sure what we gain.  If you have a strong opinion I'll do it.

Reply:
No super strong opinion. Was just an idea.